EnoCore/Models/Json/CheckerResultMessage.cs

```diff
@@ -1,13 +1,11 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using System.Text.Json.Serialization;
+﻿using System.Text.Json.Serialization;
 
 namespace EnoCore.Models.Json
 {
     public class CheckerResultMessage
     {
         [JsonPropertyName("result")]
         public string Result { get; set; } = default!;
+        public string Message { get; set; } = default!;

```

Comment:
Did you update the spec? Aren't you missing an `[JsonPropertyName("message")]`?

Reply:
its not in the json right now, therefore i did not update the spec and also did not add the tag. It will be in the next commit